notebooks/file_calibration.ipynb

```diff
@@ -61,6 +61,7 @@
     "from hera_notebook_templates.data import DATA_PATH as HNBT_DATA\n",
```

Comment:
<p>If you want to add this plot, you need to modify the list of plots (and links) at the top of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

Reply:
Added a link